schemas/service.yml

```diff
@@ -105,3 +105,13 @@
 
         This id normally changes across restarts, but `service.id` does not.
       example: 8a4f500f
+
+    - name: address
+      level: extended
+      type: keyword
+      short: Address of this service.
+      description: > 
+        Address where this service is being run from.
+
+        This could be a URL or a filepath.
+      example: localhost:8000
```

Comment:
With my [rfc1738](https://datatracker.ietf.org/doc/html/rfc1738#section-5) glasses 👓 on this example doesn't appear to be fully formed URL. Perhaps the definition needs clarified like the value must be a URI, absolute file path, or network address (ipv4:port or [ipv6]:port).

Reply:
Agree with Andrew, it would be safer to mention a network address or resource path (sockets).